docs/content/_index.md

```diff
@@ -1,42 +1,70 @@
 ---
-title: "Infection Monkey Documentation Hub"
+title: "Infection Monkey Documentation"
 date: 2020-05-26T18:15:37+03:00
 draft: false
 ---
 
-# Infection Monkey documentation hub
+# Infection Monkey documentation
 
-{{< homepage_shortcuts >}}
+
+![Infection Monkey Documentation Logo](/images/monkey-teacher.svg?height=300px
+"Infection Monkey Documentation Logo")
 
 ## What is Guardicore Infection Monkey?
+Infection Monkey is an open-source adversary emulation platform that helps you
+improve your security posture using empirical data. Monkey uses various methods
+to self-propagate across a network and reports its activities to a centralized
+command and control server known as the Monkey Island. 🐵🏝️ You know, like
+malware, but ✨safe.✨
+
+## How does it work?
+
+Infection Monkey is comprised of two components:
+
+* **Agent** - A configurable network worm that can infect machines, steal
+  data, and deliver payloads.
+* **Monkey Island** - A command and control server used to control and
+  visualize the Infection Monkey's progress throughout the simulation.
+
+### Build up your malware antibodies 💉
 
-The Infection Monkey is an open-source breach and attack simulation tool for
-testing a data center's resiliency to perimeter breaches and internal server
-infection. Infection Monkey will help you validate existing security solutions
-and will provide a view of the internal network from an attacker's perspective.
+![Malware Vaccine](/images/monkey-iv.png "Malware Vaccine")
```

Comment:
Why does this image have a drop shadow but the logo does not? I think this would look better without the shadow

Reply:
My guess is that it's because the logo is an SVG and this is a PNG. All of the PNGs are rendered the same way.

Reply:
I also had the same assumption